### PR TITLE
Made small styling changes with more to come

### PR DESCRIPTION
**Pull request by bmgdevelopment**

### CODE REVIEW

File: components/Search.js [hunk 1]
```diff
@@ -29,9 +32,14 @@ export default function Search({ data, func, attrs, placeholder }) {
...
   return (
-    <div>
-      <input value={inputValue} onChange={handleChange} placeholder={placeholder || 'Search'} />
-      <button onClick={handleClick}>Clear</button>
+    <>
+    <div class="d-flex justify-content-center" style={{alignItems: 'center', marginBottom: '20px'}}>
+      <input style={{border: '1px solid lightgray', borderRadius: '30px', width: '40vh', height: '35px', marginLeft: '10px', paddingLeft: '20px'}} value={inputValue} onChange={handleChange} placeholder={placeholder || 'Search'} />
+      <p onClick={handleClick} style={{margin: '10px', color: 'red'}}>Reset</p>
     </div>
+      {/* <button onClick={handleClick}>Clear</button> */}
```
Comment (drteresavasquez) [inline components/Search.js]: We can get rid of this

File: components/Search.js [hunk 2]
```diff
@@ -29,9 +32,14 @@ export default function Search({ data, func, attrs, placeholder }) {
   };
 
   return (
-    <div>
-      <input value={inputValue} onChange={handleChange} placeholder={placeholder || 'Search'} />
-      <button onClick={handleClick}>Clear</button>
+    <>
+    <div class="d-flex justify-content-center" style={{alignItems: 'center', marginBottom: '20px'}}>
+      <input style={{border: '1px solid lightgray', borderRadius: '30px', width: '40vh', height: '35px', marginLeft: '10px', paddingLeft: '20px'}} value={inputValue} onChange={handleChange} placeholder={placeholder || 'Search'} />
+      <p onClick={handleClick} style={{margin: '10px', color: 'red'}}>Reset</p>
```
Comment (drteresavasquez) [inline components/Search.js]: Let's add a `style={{cursor: 'pointer'}}` to this p tag

**Review by drteresavasquez — CHANGES_REQUESTED** [review]
A couple of small changes

File: components/Search.js (hunk 2 above)
Comment (bmgdevelopment) [reply components/Search.js]: On it!

File: components/Search.js (hunk 1 above)
Comment (bmgdevelopment) [reply components/Search.js]: Done

**Review by drteresavasquez — APPROVED** [review]
👍🏼  looks good!

**Pull request merged**